nesta/core/schemas/tier_1/datasets/arxiv.json

```diff
@@ -0,0 +1,28 @@
+{
+    "entity_type": "article",
+    "tier0_to_tier1": {
```

Comment:
Here I'd use a symbol as a separator like `tier0,tier1` or `tier0:tier1`.

Reply:
are you suggesting that I should use a comma or colon in a field name? 😲again, this will also removed in daps2 (in favour of implicit mapping of raw field names to standardised names) so it won't burden us for too long.

Reply:
yes, :) I'd favour `tier0:tier1`
OK.